src/AuthorizedSocketConnection.js

```diff
@@ -61,11 +61,23 @@ export default class AuthorizedSocketConnection<TContext, TCredentials> {
     return this.config.credentialsManager.getCredentials();
   }
 
-  isAuthorized = (data: any) =>
-    !!(
-      this.config.credentialsManager.isAuthenticated() &&
-      this.config.hasPermission(data, this.getCredentials())
+  isAuthorized = (data: any) => {
+    const isAuthenticated = this.config.credentialsManager.isAuthenticated();
+    const credentials = this.getCredentials();
+
+    const isAuthorized = !!(
+      isAuthenticated && this.config.hasPermission(data, credentials)
     );
+    if (!isAuthorized) {
+      this.log('info', 'unauthorized', {
```

Comment:
I was somewhat reticent to log out credentials but I guess meh?

Reply:
The credentials are just the dictionary, right? Not the actual token.

Reply:
Ah right.